sdk/core/core-tracing/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure SDK Core client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure SDK Core in some common scenarios.
+
+| **File Name**                   | **Description** |
+| ------------------------------- | --------------- |
+| [basicTracing.js][basictracing] | Uses            |
```

Comment:
Interesting niche scenario here - the description stops after `Uses`... is it because the next character is `@`? @willmtemple as an FYI

Reply:
I wrapped `@azure/core-tracing` in ``` and that seemed to work